ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -682,28 +721,64 @@ const NetworksForm = ({
           onChange={(value) => {
             setIsEditing(true);
             setChainId(value);
+            autoSuggestTicker(value);
           }}
           titleText={t('chainId')}
           value={chainId}
           disabled={viewOnly}
           tooltipText={viewOnly ? null : t('networkSettingsChainIdDescription')}
         />
-        <FormField
-          warning={warnings.ticker?.msg || ''}
-          warningProps={{
-            'data-testid': 'network-form-ticker-warning',
-            style: {
-              color: 'var(--color-warning-default)',
-            },
-          }}
-          onChange={(value) => {
+        <FormTextField
+          data-testid="network-form-ticker"
+          helpText={
+            suggestedTicker ? (
+              <Text
+                as="span"
+                variant={TextVariant.bodySm}
+                color={TextColor.textDefault}
+              >
+                {t('suggestedTokenSymbol')}{' '}
+                <Text
+                  as="span"
```

Comment:
Let's make this a `button` to ensure keyboard accessibility.  Let's also ensure it has the `pointer` cursor.